PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--- Use a prefix like [TASK], [BUGFIX], [DOC] or [CGL] and provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please try to test the code in multiple browsers and also on a mobile device -->
```

Comment:
Similar to the "Your environment" section above, this often doesn't make sense, e.g. if the PR only touches backend code. Maybe it could be made a bit more generic?
> \<!--- Please try to test the code in multiple situations and with different settings that could influence it -->